ArchiSteamFarm/Steam/Integration/SteamChatMessage.cs

```diff
@@ -86,6 +87,10 @@ internal static class SteamChatMessage {
 
 					// Check if we reached the limit for one message
 					if (messagePartBytes + NewlineWeight + ReservedEscapeMessageBytes > maxMessageBytes) {
```

Comment:
This block isn't covered by any unit test.

Reply:
Not critical, can be done post-PR if somebody has time.